lib/oci_utils/kvm/virt.py

```diff
@@ -549,11 +548,14 @@ def create(**kargs):
         _disk_virt_install_args = _root_disk
     else:
         _disk_virt_install_args = 'pool=%s,size=%d' % (kargs['pool'], kargs['disk_size'])
-
-    args = [SUDO_CMD, '/usr/bin/virt-install',
-            '--name', kargs['name'],
-            '--cpu', 'host',
-            '--disk', _disk_virt_install_args]
+    args = []
+    if os.getuid() == 0:
+        args.append(SUDO_CMD)
```

Comment:
shouldn;t this sound:
if os.getuid() != 0

Reply:
good catch. 
Indeed , should be '!='